independent-publisher-connectors/Notion/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@sekharmalla The settings.json file is not required, please remove it from the PR.

Reply:
Have done it thanks.